eth/fetcher/block_fetcher.go

```diff
@@ -483,7 +483,7 @@ func (f *BlockFetcher) loop() {
 							select {
 							case res := <-resCh:
 								res.Done <- nil
-								f.FilterHeaders(peer, *res.Res.(*eth.BlockHeadersRequest), time.Now().Add(res.Time))
+								f.FilterHeaders(peer, *res.Res.(*eth.BlockHeadersRequest), time.Now())
```

Comment:
It shouldI think it originally wanted to be `Sub` instead of `Add`, to track the time of announcement https://github.com/ethereum/go-ethereum/blob/master/eth/fetcher/block_fetcher.go#L599. But in reality, that line feels a bit wonky too, as why would I want to update the announcement time if I already track it anyway?